micropsi_integration_sdk/robot_sdk.py

```diff
@@ -386,6 +386,28 @@ def get_guiding_filter_strength(self) -> float:
         """
         return 0.0
 
+    def get_ra_acceleration_values(self) -> dict:
+        """
+        Optional, override as appropriate.
+
+        This function returns a dict of acceleration values for the robot's Recording Assistant
+        controller. The values are in m/s^2 and deg/s^2 for linear and angular acceleration
+        respectively. The values are used to limit the maximum linear and angular velocity changes
+        between two steps of the controller. By doing so, we avoid sudden shifts in the robot's
+        velocity, which could cause the robot to dangerously jerk and potentially overshoot the
+        target positions. Hence, the values are specific to the robot's acceleration and
+        deceleration capabilities.
+
+        Default values below are tested to function well with URs (UR3, UR5, UR10) and Fanucs
+        (CRX10iA, LR Mate 200iD).
```

Comment:
Fanucs (CRX-10iA/L, LR Mate 200iD/7L)
We do not have such reference `CRX-10iA` in the lab yet, and LR Mate has many short and large robots models of the same reference `LR Mate 200iC`.